src/parser/jobs/mnk/modules/RiddleOfFire.tsx

```diff
@@ -176,28 +167,14 @@ export class RiddleOfFire extends Analyser {
 		this.suggestions.add(new TieredSuggestion({
 			icon: this.data.actions.ELIXIR_FIELD.icon,
 			content: <Trans id="mnk.rof.suggestions.ogcd.content">
-				Aim to use <DataLink action="TORNADO_KICK"/>, <DataLink action="ELIXIR_FIELD"/>, and at least 1 <DataLink action="SHOULDER_TACKLE"/> during each <DataLink status="RIDDLE_OF_FIRE"/>.
+				Aim to use <DataLink action="TORNADO_KICK"/>, <DataLink action="ELIXIR_FIELD"/>, and at least 1 <DataLink action="THUNDERCLAP"/> during each <DataLink status="RIDDLE_OF_FIRE"/>.
```

Comment:
This copy expects the user to use Thunderclap in each Riddle of Fire window, but we are not tracking it anywhere or displaying how many Thunderclaps they are actually using.  We should add Thunderclap tracking to replace the removed Shoulder Tackle information.

Reply:
We should just not? Thunderclap is no damage teleport skill.

Reply:
Ah, yeah, then the copy should be fixed to not specify using Thunderclap